__tests__/find-ruby.test.ts

```diff
@@ -56,10 +56,10 @@ describe('find-ruby', () => {
 });
 
 function normalize(pathToFix: string) {
-  let p = pathToFix;
+  let p = path.normalize(pathToFix);
 
   if (os.platform() === 'win32') {
-    p = p.replace(/\//g, '\\');
+    // p = p.replace(/\//g, '\\');
```

Comment:
Can just remove this if it isn't needed.

Reply:
I'm going to change this based on Ting feedback above